src/configuration.ts

```diff
@@ -19,20 +19,35 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
-const CONFIG_FILENAME = "tslint.json";
-const DEFAULT_CONFIG = {
+export const CONFIG_FILENAME = "tslint.json";
+export const DEFAULT_CONFIG = {
     "rules": {
-        "curly": true,
-        "indent": [true, 4],
-        "no-duplicate-key": true,
+        "class-name": true,
+        "comment-format": [true, "check-space"],
+        "indent": [true, "spaces"],
         "no-duplicate-variable": true,
-        "no-empty": true,
         "no-eval": true,
+        "no-internal-module": true,
         "no-trailing-whitespace": true,
-        "no-unreachable": true,
-        "no-use-before-declare": true,
-        "quotemark": [true, "double"],
-        "semicolon": true
+        "no-var-keyword": true,
+        "one-line": [true, "check-open-brace", "check-whitespace"],
+        "quotemark": [true, "single"],
```

Comment:
meh, but it doesn't make sense to have the default config be different from our own code like this. we should discuss moving to single quotes elsewhere... we choose double to make it easier to switch back-and-forth between languages like Java to TS. many others, including the compiler team, use double quotes for the same reason.